assets/src/edit-story/elements/text/preview.js

```diff
@@ -57,8 +57,8 @@ function TextPreview( {
 		fontWeight: fontWeight ? fontWeight : null,
 		background: backgroundColor,
 		margin: 0,
-		color:color,
-		lineHeight:lineHeight,
+		color,
```

Comment:
The linter kicked off about this. I did not make this change for the record.

Reply:
Yep, it's odd, the linter is auto-fixing it to `color: color` instead of using `color`. cc @swissspidy

Reply:
No, I run the auto-fixer locally. But I dont know how this code was in the code base.

Reply:
Yep, that's what I wanted to mention  -- that it's locally auto-fixing it to `color: color` etc. in this one place in the code when running auto-fixer (at least in my local!). Probably some configuration issue.

Reply:
Might not be an issue anymore with Prettier, where these things are more separated.